resources/views/books/create.blade.php

```diff
@@ -2,9 +2,8 @@
 
 @section('content')
     <div class="container">
-        <h1>Kitap Ekle</h1>
+        <h1>{{ isset($book) ? 'Kitabı Düzenle' : 'Kitap Ekle' }}</h1>
```

Comment:
create blade'inde neden böyle bir şey var? ya view isimlendirmesinde sıkıntı var. ya da kodda var. hem update hem create için kullanacak şekilde ayarladıysan ismini "form" vs. yapman daha doğru olur. eğer ayrı ayrı güncelleme ve create view dosyası varsa o zaman burada "düzenle" için case olmamalı.

Reply:
dosyanın tümü genel bir update'e kaymış nedense, bu dediğime göre create ve edit dosyalarını bir tekrar gözden geçir.